### DISCUSSION TITLE
is CUDA available in ROCm platform?

### DISCUSSION THREAD

**Discussion by opticaltak43** [CederGroupHub/chgnet#135] [Q&A]
hi，I have installed chgnet and torch-rocm, and the cuda is available in torch, but the CHGNET model can not be loaded. it seems that nvidia-smi is needed.

**Comment by bowen-bd** [comment]
We have not tested this.
In principle, this should be OK.
Can you please locate where the bug is?
If you can resolve the compatibility, can you make a PR for this?

Thanks
